lib/findable.rb

```diff
@@ -1,22 +1,26 @@
 module Dor::Assembly
   module Findable
+    attr_reader :folder_style
```

Comment:
So I am not sure this buys us a whole lot -- it can eliminate one method in findable, but I still need the others for various reasons (like searching through multiple possible root directories to find the base content, like /dor/assembly and /dor/workspace and /data/hydrus-files, etc.).  I also need a way to find the base content folder itself, those find_xxx methods look for specific files.  So the only method I can see to update would be path_to_metadata_file and path_to_content_file and then I could path_to_metadata_folder.